README.md

```diff
@@ -66,20 +66,25 @@ In order to use the SDK installed via vcpkg with CMake, you can use the toolchai
 > cmake --build [build directory]
 ```
 
-The **entry point** for most scenarios when using the SDK will be a top-level client type corresponding to the Azure service you want to talk to. For example, sending requests to blob storage can be done via the `Azure::Storage::Blobs::BlobClient` API.
+#### Using the SDK within your Application
 
-All the Azure C++ SDK headers needed to be included are located within the `<azure>` folder, with sub-folders corresponding to each service. Similarly, all types and APIs can be found within the `Azure::` namespace. For example, to use functionality form `Azure::Core`, include the following header at the beginning of your application `#include <azure/core.hpp>`.
+The **entry point** for most scenarios when using the SDK will be a top-level client type corresponding to the Azure service you want to talk to. For example, sending requests to blob storage can be done via the `Azure::Storage::Blobs::BlobClient` API. All APIs on the client type send HTTP requests to the cloud service and return back an HTTP `Response<T>`.
+
+All the Azure C++ SDK headers needed to be included are located within the `<azure>` folder, with sub-folders corresponding to each service. Similarly, all types and APIs can be found within the `Azure::` namespace. For example, to use functionality from `Azure::Core`, include the following header at the beginning of your application `#include <azure/core.hpp>`.
 
 Here's an example application to help you get started:
 
 ```C++
 #include <iostream>
+
+// Include the necessary SDK headers
 #include <azure/core.hpp>
 #include <azure/storage/blobs.hpp>
 
+// Use the appropriate namespace using directives
 using namespace Azure::Storage::Blobs;
 
-// Get the required connection string/key from an environment variable or Azure Key Vault.
+// Get the required connection string/key from an environment variable or Azure KeyVault.
 std::string GetConnectionString();

```

Comment:
Is this a TODO or something?
We actually discourage the use of connection strings; can we change this?

Reply:
@JeffreyRichter how do you recommend we construct a blob client, if not using a connection string, as a copy-pasteable sample for customers?

Reply:
> We actually discourage the use of connection strings

Why?  For Storage, that's super easy to get the connection string from the Azure portal and use it.  Is it even easier to set the shareKeyCredential?

Reply:
Connection strings usually combine multiple things together that should not be combined like endpoint, username & password. We want to encourage people to roll over passwords periodically while their service keeps running but if they are in a connection string, then they need a whole new connection string and there is no way to atomically update an endpoint, username, and password.

In addition, there are many reasons why distributed systems fail including bad endpoints and bad credentials. So, customers can better diagnose issues if they are responsible for passing these things explicitly – it is the cost of entry for building a distributed system.

Connection strings also have the potential for lots of problems like are they case sensitive? What does it mean if multiple credentials appear in the string? Will customers embed connection strings in their source code and check that in to github? (connections strings make this bad practice too easy).

Azure SDKs policy is to NOT support connection strings unless the portal/tooling does offer them. In this case, we support them for "get up and running fast" scenario but we still discourage their use for real production.

It's even in our C++ guidelines: https://azure.github.io/azure-sdk/cpp_introduction.html#cpp-authentication